lib/loomio_i18n.rb

```diff
@@ -1,29 +1,23 @@
 module Loomio
-  module I18n
+  class I18n
+    require 'yaml'
+    config = YAML.load_file("config/loomio_i18n.yml")['loomio_i18n']
 
-    SELECTABLE_LOCALES = %i( en an be-BY bg-BG ca cs zh-TW da de eo es el fr id it he hu ja ko ml nl-NL pt-BR ro sr sr-RS sk sv vi tr uk )
+    def self.symbolize(object)
```

Comment:
Not totally sure you need this method to symbolize everything.

Would something like this do the job?

```
module Loomio
  class I18n
    require 'yaml'
    config = YAML.load_file("config/loomio_i18n.yml")['loomio_i18n']

    SELECTABLE_LOCALES = Array(config['selectable_locales']).map(&:to_sym)
    DETECTABLE_LOCALES = Array(config['additional_detectable_locales']).map(&:to_sym) + SELECTABLE_LOCALES

    RTL_LOCALES  = Array(config['rtl_locales']).map(&:to_sym)
    TEST_LOCALES = Array(config['test_locales']).map(&:to_sym)

    FALLBACKS = Hash(config['fallbacks']).with_indifferent_access

  end
end
```